### PR TITLE
B2B sample app improvements

### PR DESCRIPTION
**Pull request by sadilchamishka**

### CODE REVIEW

File: b2b-sample/config.json [hunk 1]
```diff
@@ -32,13 +24,13 @@
...
-            "UserStore" : "DEFAULT"
+            "UserStore" : "PRIMARY"
         }
     },
     "BusinessAdminAppConfig": {
         "AuthorizationConfig": {
-            "ClientId": "<CLIENT ID OF THE BUSINESS ADMIN APP>",
-            "ClientSecret": "<CLIENT SECRET OF THE BUSINESS ADMIN APP>"
+            "ClientId": "kklQ7LcGDHPSLabRfEzaNkoykmQa",
+            "ClientSecret": "_elOlfc_XVfaQHSe0zblO_f37KjFStCtozjH_ZP9SHoa"
```
Comment (AnuradhaSK) [inline b2b-sample/config.json:33]: keep them as placeholders

File: b2b-sample/config.json [hunk 2]
```diff
@@ -29,8 +29,8 @@
     },
     "BusinessAdminAppConfig": {
         "AuthorizationConfig": {
-            "ClientId": "kklQ7LcGDHPSLabRfEzaNkoykmQa",
-            "ClientSecret": "_elOlfc_XVfaQHSe0zblO_f37KjFStCtozjH_ZP9SHoa"
+            "ClientId": "<CLIENT ID OF THE BUSINESS ADMIN APP>",
+            "ClientSecret": "<CLIENT SECRET OF THE BUSINESS ADMIN APP>"
```
Comment (sadilchamishka) [inline b2b-sample/config.json:33]: A comment address.

**Review by AnuradhaSK — APPROVED** [review]

File: b2b-sample/libs/business-app/data-access/data-access-controller/src/lib/controller/controllerSwitchOrg/controllerDecodeSwitchOrg.ts [hunk 3]
```diff
@@ -25,16 +25,13 @@ import config from "../../../../../../../../config.json";
...
-            return config.CommonConfig.ApplicationConfig.SampleOrganization[0].id;
-        } else {
+        if (token.user.org_id) {
 
             return token.user.org_id;
-        }
+        } 
+        
+        return config.CommonConfig.ApplicationConfig.SampleOrganization[0].id;
+
```
Comment (Achintha444) [inline b2b-sample/libs/business-app/data-access/data-access-controller/src/lib/controller/controllerSwitchOrg/controllerDecodeSwitchOrg.ts:34]: Remove this line

File: b2b-sample/apps/business-admin-app/pages/api/auth/[...nextauth].ts [hunk 4]
```diff
@@ -51,23 +51,18 @@ const wso2ISProvider = (req: NextApiRequest, res: NextApiResponse) => NextAuth(r
...
-                session.idToken = orgSession.id_token;
-                session.scope = orgSession.scope;
-                session.refreshToken = orgSession.refresh_token;
+            } else {
+                session.error = false;
                 session.expires = false;
-                session.userId = getLoggedUserId(session.idToken);
+                session.accessToken = token.accessToken;
+                session.scope = token.scope;
+                //session.userId = getLoggedUserId(session.idToken);
```
Comment (Achintha444) [inline b2b-sample/apps/business-admin-app/pages/api/auth/[...nextauth].ts:62]: Remove this line

**Pull request merged**